tests/staggered_gsmear_test_utils.h

```diff
@@ -5,17 +5,19 @@
 #include <cstdlib>
 #include <cstring>
 #include <algorithm>
+#include <assert.h>
```

Comment:
Mild preference for `cassert` instead of `assert.h` since the former is the C++ variant.  If changing things anyway, can make this change (here and wherever else you are touching).

Reply:
https://github.com/lattice/quda/commit/a45cb662ec707800a2fe86da8600956208666204